platforms/software/dependency-management/src/main/java/org/gradle/api/internal/catalog/DefaultVersionCatalogBuilder.java

```diff
@@ -180,7 +179,7 @@ private DefaultVersionCatalog doBuild() {
         maybeImportCatalogs();
         if (!aliasesInProgress.isEmpty()) {
             String alias = aliasesInProgress.iterator().next();
-            throw throwVersionCatalogProblemException(getProblemService().create(builder ->
+            throw throwVersionCatalogProblemException(getProblemService(), getProblemService().internalProblemReporter().create(builder ->
```

Comment:
❌ we should have a `getInternalReporter()` for `getProblemService().internalProblemReporter()`

Reply:
The suggestion is to rename `internalProblemReporter()` to `getInternalReporter()`, right?